gpiozero/output_devices.py

```diff
@@ -324,7 +384,11 @@ class LED(DigitalOutputDevice):
         See :doc:`api_pins` for more information (this is an advanced feature
         which most users can ignore).
     """
-    pass
+    def __init__(self, pin=None, active_high=True, initial_value=False, pin_factory=None):
+        self._blink_thread = None
+        self._controller = None
+        super(LED, self).__init__(pin, active_high, initial_value, pin_factory=pin_factory)
```

Comment:
Why is this change needed, when `DigitalOutputDevice`'s `__init__` already sets `_blink_thread` and `_controller` to None? :man_shrugging: